src/components/LoadingSpinner.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
-import { LoadingIcon } from '../components/SvgIcons';
+import { css } from 'glamor';
+import { LoadingIcon, NotFoundIcon } from '../components/SvgIcons';
 
-export default function LoadingSpinner() {
+export default function LoadingSpinner({ error = false }) {
+  const svgSizeStyle = { width: '5rem', height: '5rem' };
   return (
-    <div className="loading-spinner">
-      <LoadingIcon className="loading-spinner-svg" />
+    <div
+      {...css({
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100%',
+      })}
+    >
+      {error ? (
+        <NotFoundIcon {...css(svgSizeStyle)} />
+      ) : (
+        <LoadingIcon {...css(svgSizeStyle)} />
+      )}
```

Comment:
I think I see why you did it (D.R.Y. styles) but I don't dig using a single `LoadingSpinner` component for both the error state and the loading spinner. I'd probably just fork it into two components, even if there's some redundant CSS. 😄

Reply:
That's also what I was thinking and was not sure when coping with this part. Thanks for pointing out.